core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/util/UserCoreUtil.java

```diff
@@ -120,6 +121,21 @@
         return (String[]) h.toArray(new String[h.size()]);
     }
 
+    /**
+     * Convert a map of lists to a list with unique elements.
+     *
+     * @param mapOfLists Map of lists.
+     * @return list with unique elements.
+     */
+    public static Set<String> getList(Map<String, List<String>> mapOfLists) {
```

Comment:
Is this a common requirement to use as a Util? Better use as a private method.

Reply:
Fixed and updated the PR.